docs/en/observability/synthetics-manage-monitors.asciidoc

```diff
@@ -1,37 +1,37 @@
-[[synthetics-manage-monitors]]
-= Manage browser monitors
+// [[synthetics-get-started-project]]
```

Comment:
Is there a reason this is all commented out rather than being deleted?

Reply:
I left this here because I haven't found a place for it in the new structure yet.

Reply:
@andrewvc do you think the content on the [Manage browser monitors](https://www.elastic.co/guide/en/observability/current/synthetics-manage-monitors.html) page is helpful? I haven't found the right place for this content yet. Maybe it is its own page at the highest level in the table of contents (maybe after "Scripting browser monitors" and "Configure lightweight monitors")?

Reply:
That content feels like it's largely there for the sake of completeness, though I certainly don't _mind_ it. That said, I agree it's in the wrong place from a hierarchy standpoint, it applies to both lightweight and browser monitors. So yes, a top-level item would make more sense.

![image](https://user-images.githubusercontent.com/131427/202810163-2e208d08-6627-4da6-8ed5-e3e4a12be561.png)